### ISSUE TITLE
Unable to install adldap2

### ISSUE THREAD

**Issue by FMWK** [grokability/snipe-it#10317]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

This started out while trying to run `php artisan snipeit:ldap-sync` which returned `Use of undefined constant LDAP_OPT_PROTOCOL_VERSION`

After a search, this led me to the php_ldap extension being disabled. I have since checked this is enabled in the assigned PHP ini file - this is verifiable in the phpinfo() output and also the IIS "Enabled Extensions" section. 

I've also verified that the ini file being loaded by PHP is _C:\Program Files\PHP\v7.4\php.ini_ via CMD

Worth noting that I cannot see LDAP listed in `php -m`

Running `composer check-platform-reqs` shows that the **ext-ldap** package is missing. I am unable to see what I've missed in the setup for the ldap module to not be recognised as enabled?

- Server has been restarted
- IIS instance has been restarted

### Reproduction steps

-

### Expected behavior

Running `composer update` returns:

` Problem 1
    - Root composer.json requires PHP extension ext-fileinfo * but it is missing from your system. Install or enable PHP's fileinfo extension.
  Problem 2
    - Root composer.json requires league/flysystem 1.1.4 -> satisfiable by league/flysystem[1.1.4].
    - league/flysystem 1.1.4 requires ext-fileinfo * -> it is missing from your system. Install or enable PHP's fileinfo extension.
  Problem 3
    - adldap2/adldap2[v10.2.0, ..., v10.3.3] require ext-ldap * -> it is missing from your system. Install or enable PHP's ldap extension.
    - Root composer.json requires adldap2/adldap2 ^10.2 -> satisfiable by adldap2/adldap2[v10.2.0, ..., v10.3.3].

To enable extensions, verify that they are enabled in your .ini files:
    - C:\Program Files\PHP\v7.4\php.ini
You can also run `php --ini` inside terminal to see which files are used by PHP in CLI mode.`

### Screenshots

![image](https://user-images.githubusercontent.com/19735790/142016021-6662c5b6-73ec-4bfe-9929-be1cc3418fba.png)

### Snipe-IT Version

v5.3.1

### Operating System

Windows 2016 Datacenter

### Web Server

IIS

### PHP Version

7.4.13

### Error messages

```shell
[PHP Modules]
bcmath
calendar
Core
ctype
curl
date
dom
exif
filter
gd
gettext
hash
iconv
imap
json
libxml
mbstring
mysqli
mysqlnd
openssl
pcre
PDO
pdo_mysql
pdo_sqlite
Phar
readline
Reflection
session
SimpleXML
soap
SPL
standard
tidy
tokenizer
xml
xmlreader
xmlrpc
xmlwriter
zip
zlib
```

### Additional context

Fresh install from manual installation

**Comment by FMWK** [comment]
One, new, notable thing is that the "LDAP Sync" button on the website completes the sync without issue.

This narrows it down to it being the version/location of PHP that the CMD CLI is using/running, but still very unsure as it states the module file being loaded is the same one that the SnipeIT web service is utilising!

**Comment by snipe** [comment]
Are you still having issues with the latest version?

**Comment by FMWK** [comment]
@snipe Hi!

Running `php upgrade.php` after a `git pull` is returning 

```
--------------------- !! ERROR !! ----------------------
✘ MISSING PHP EXTENSION: fileinfo
✘ MISSING PHP EXTENSION: ldap
```

so the original issue still seems to be persisting. PHPINFO() shows ldap and fileinfo in the list of extensions.

![image](https://user-images.githubusercontent.com/19735790/170112966-506a8743-5490-4cba-8893-e395f1a2cd4e.png)

**Comment by snipe** [comment]
Command line PHP and web-server PHP can sometimes be two different versions. Try running `php -m` from the command line to see what extensions are installed on your cli version.

**Comment by snipe** [comment]
Fileinfo is going to be the more important one here tho. It's been a requirement for Snipe-IT for a very long time (years).

**Comment by snipe** [comment]
This is pretty definitely a server configuration issue, not an issue with the upgrader or the Snipe-IT app itself. You'll need to get the fileinfo extension part handled, but you can always try a manual upgrade. 

```
git pull
composer install --no--dev --prefer-source
composer dump-autoload
php artisan migrate
```

The upgrader just adds some additional checks for libraries and directory permissions.

**Comment by FMWK** [comment]
@snipe Okay. I found the issue in the end after a random thought came to me when running `php --ini` and similar commands. Even though I had everything pointing at the `C:\Program Files\PHP\v7.4\` directory (and the above command referenced that too), I thought to double check the environment variables and what was in `PATH` - in here I found two difference instances of PHP directories, one being the above mentioned, and the other being the one bundled into the IISExpress directory.

Removing the `C:\Program Files\PHP\v7.4\` PATH and re-running `php --ini` showed the now changed PHP directory, all the modules loaded via `php -m` and `php upgrade.php` then ran without issue! 

Can only assume that the duplicate PHP entry, albeit different dirs, was causing a conflict along the lines. (Both php.ini instances has ldap and fileinfo enabled, too)
